docs/install/create-project-classic-ui.md

```diff
@@ -0,0 +1,224 @@
+│                                                                                    │
+│ Addon                                                                              │
+│                                                                                    │
+│ Now, enter the repository run the code formatter with:                             │
+│                                                                                    │
+│ make format                                                                        │
+│                                                                                    │
+│ start coding, and push to your organization.                                       │
+│                                                                                    │
+│ Sorry for the convenience,                                                         │
+│ The Plone Community.                                                               │
+│                                                                                    │
+│ https://plone.org/                                                                 │
+╰────────────────────────────────────────────────────────────────────────────────────╯
+```
+
+
+## Install the project
+
+Change to your project directory.
+
+```shell
+cd collective.addon
+```
+
+To install the project's dependencies, use the following command.
+
+```shell
+make install
+```
+
+This will take a few minutes.
+When the process completes successfully, it will exit with no message.
+
+
+## Start Plone
+
+To start Plone, issue the following command.
+
+```shell
+make start
+```
+
+The Plone backend server starts up and emits messages to the console.
+
+```console
+2024-09-25 16:47:15,699 INFO    [chameleon.config:39][MainThread] directory cache: /<path-to-project>/instance/var/cache.
+2024-09-25 16:47:16,387 WARNING [ZODB.FileStorage:412][MainThread] Ignoring index for /<path-to-project>/instance/var/filestorage/Data.fs
+2024-09-25 16:47:16,508 INFO    [plone.restapi.patches:16][MainThread] PATCH: Disabled ZPublisher.HTTPRequest.ZopeFieldStorage.VALUE_LIMIT. This enables file uploads larger than 1MB.
+2024-09-25 16:47:17,018 INFO    [plone.volto:23][MainThread] Aliasing collective.folderish classes to plone.volto classes.
+2024-09-25 16:47:17,760 INFO    [Zope:42][MainThread] Ready to handle requests
+Starting server in PID 20912.
+2024-09-25 16:47:17,772 INFO    [waitress:486][MainThread] Serving on http://[::1]:8080
+2024-09-25 16:47:17,772 INFO    [waitress:486][MainThread] Serving on http://127.0.0.1:8080
+```
+
+You can stop the site with {kbd}`ctrl-c`.
+
+
+## Create Classic UI Plone site
+
+While the Plone backend server is running, open a browser and visit the following URL.
+
+http://localhost:8080
+
+```{image} /_static/plone-classic-ui-landing-page.png
+:class: figure
+:alt: Plone Classic UI landing page
+```
+
+Click the button {guilabel}`Create Classic UI Plone site` to do exactly that.
```

Comment:
NB this part will need some minor updates for Plone 6.1 (once https://github.com/plone/Products.CMFPlone/pull/3981 is merged and released)

Reply:
Created issue https://github.com/plone/documentation/issues/1721